INSTALL.md

```diff
@@ -44,7 +45,7 @@ docker run -it --name elf-simulations-dev --rm --volume $(pwd):/app/ --net=host
 ## Apeworks and Contract Integration
 
 NOTE: The Hyperdrive solidity implementation is currently under security review, and thus is not available publicly.
-The following instructions will not work for anyone who is not a member of Element Finance.
+The following instructions will not work for anyone who is not a member of Delv.
```

Comment:
I'd prefer these to be their own PR, but I'll accept this for now

Reply:
har har